strategy-explorer/bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "strategy-explorer",
+  "homepage": "https://github.com/shans/arcs",
+  "authors": [
+    "shans <shane.stephens@gmail.com>"
+  ],
+  "description": "",
+  "main": "",
+  "license": "MIT",
```

Comment:
Can you delete everything except ignore and dependencies? At least get rid of license, authors and homepage.